debian/tests/test

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+HELP_MSG="The script requires exactly 1 parameter"
```

Comment:
Which ones :p (yes I know, but as we only have 2 options, better to write them :p)